Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -33,10 +33,10 @@ public typealias PurchaseResultData = (transaction: StoreTransaction?,
 /**
  Completion block for ``Purchases/purchase(product:completion:)``
  */
-public typealias PurchaseCompletedBlock = @MainActor @Sendable (StoreTransaction?,
-                                                                CustomerInfo?,
-                                                                PublicError?,
-                                                                Bool) -> Void
+public typealias PurchaseCompletedBlock =  @MainActor @Sendable (StoreTransaction?,
```

Comment:
This one keeps the `@MainActor`. I'm guessing it's not used anymore in this v4. If so, perhaps we should remove it completely to prevent potential future errors if we need to make a new release of v4?

Reply:
I think the concern around that one was about making further changes to the API - like, if your code relies on this being `@MainActor`, you might have broken compilation after upgrading. 

I mean, the original problem was to add `@MainActor` in a non-minor anyway, but kinda don't want to repeat the same issue twice

Reply:
might as well kill the extra space ` ` so it doesn't even show up in the diff tho 😅